gemrb/core/Store.cpp

```diff
@@ -339,6 +339,14 @@ void Store::RemoveItem(const STOItem *itm)
 	}
 }
 
+void Store::RemoveItemByName(const ResRef &itemname)
+{
+	unsigned int idx = this->FindItem(itemname, false);
+	if (idx == (unsigned int) -1) return;
+	STOItem *si = this->GetItem(idx, false);
```

Comment:
should be const

Reply:
But **RemoveItem** is not a const it will not work. I have to define RemoveItem as const

Reply:
Why? It's just about this STOItem that you don't modify and RemoveItem also doesn't.